site/gulpfile.js

```diff
@@ -98,32 +102,34 @@ gulp.task('swag-img:optimize', () => {
 
 gulp.task('swag-img', gulp.series('swag-img:clean', 'swag-img:download' , 'swag-img:optimize'));
 
-gulp.task('clean', () => {
-    return del([
-        './rev-manifest.json',
-        'dist/assets/css/*',
-        'dist/assets/js/*',
-        'dist/assets/swag-img/*'
-    ]);
-});
+gulp.task('clean:styl', () => del('dist/assets/css/*'));
+gulp.task('clean:js', () => del('dist/assets/js/*'));
+gulp.task('clean:assets', gulp.parallel('clean:styl', 'clean:js'));
 
 gulp.task('cachebust', cb => {
+    const basePath = 'dist/assets';
+    const revManifestConfig = {
+        base: basePath,
+    };
     const bustedFiles = [
         'dist/assets/css/*',
         'dist/assets/js/*',
         'dist/assets/swag-img/*'
     ];
 
-    return gulp.src(bustedFiles, {base: 'dist/assets/'})
-        .pipe(cachebust())
-        .pipe(cacheClean())
-        .pipe(gulp.dest('dist/assets'))
-        .pipe(cachebust.manifest({
-            base: 'dist/assets',
-            merge: true
-        }))
-        .pipe(gulp.dest('dist/assets/'))
+    return gulp.src(bustedFiles, {base: basePath})
+        .pipe(CACHEBUST ? cachebust() : noop())
```

Comment:
While this logic works, imo it's really unreadable. I think we should just do something like
```js
if (!CACHEBUST) {
	// glob is a subdep which we can hoist to the root, just use the same version :)
    const glob = require('glob');
	const {writeFileSync} = require('fs');
	// glob takes the glob as `{glob1,glob2,glob3}`
    const fileList = `{${bustedFiles.join(',')}}`;
	// this can be cleaned up a either using promises or glob.sync
    glob(fileList, (err, files) => {
		const json = files
			.map(file => file.replace('dist/assets/', ''))
			.reduce((state, file) => { state[file] = file; return state; }, {});
		fs.writeFileSync('filename', JSON.stringify(json, null, 2));
        cb();
    });
    return;
}

// Normal case
```